reasoner_pydantic/results.py

```diff
@@ -46,6 +50,25 @@ class Config:
         extra = "allow"
 
 
+class Analysis(BaseModel):
+    """Result analysis"""
+
+    source: InformationResource = Field(..., title="list of node bindings")
```

Comment:
For consistency, it would be a good idea to label this field "resource" instead. That way, it uses the same nomenclature as kedge sources.

Reply:
Agreed.